script/build

```diff
@@ -7,12 +7,14 @@ rm -rf log/test.log
 touch log/development.log
 touch log/test.log
 
-if [ $(uname) = "Darwin" ]; then
-  echo "---- Making sure docker-machine is running"
-  docker-machine start default || true
+if ! command -v docker > /dev/null; then
```

Comment:
This is for linux environments which already have docker without a docker-machine, and also if docker already available just run it